operations/rack_resiliency/rack_resiliency.md

```diff
@@ -0,0 +1,490 @@
+
+**Note:** The policy has been enabled to allow running replicas on the same rack if other racks are not available due to a failure. This ensures that the number of replicas during a failure remains the same.
+
+### 4 Monitor failures and ensure critical services are available across zones
+
+The Resiliency Monitoring Service (RMS) is designed to continuously monitor the health and availability of CSM critical services and CSM management nodes.
+
+RMS integrates with the HMNFD service and relies on its notifications to detect node-level failures in real time.
+
+#### 4.1 RMS Architecture Overview
+
+RMS operates using two primary control loops:
+
+###### 4.1.1 Control Loop
+
+This loop ensures the monitoring infrastructure is properly initialized and maintained. It performs the following tasks:
+
+* Starts the internal application server to receive notifications from HMNFD.
+* Verifies and maintains active subscriptions to HMNFD.
+* Periodically updates the status of critical services and zones when no active monitoring events are in progress.
+
+###### 4.1.2 Monitoring Loop
+
+Triggered upon receiving a notification from HMNFD, this loop performs targeted analysis and response actions, including:
+
+* Determining whether the failure is isolated to a single node or part of a rack-level failure.
+* Monitors the health and status of critical services, and logs alerts if any imbalance is detected following a failure.
+* Verifying placement and availability of management nodes, including both `Kubernetes` control plane and `Ceph` storage nodes.
+
+**Note:** Any modifications to critical services may take up to 10 minutes to reflect in CLI command outputs and to appear in the monitored list.
+
+### 5 RR CLI
+
+The CLI for interfacing with rack resiliency service is part of `craycli`. A new module (`rrs`) is added to `craycli` to support rack resiliency specific commands. The backend for the CLI which serves REST endpoints is provided as a container along with the rack resiliency monitor in the same pod.
+
+#### Commands 
+
+The following commands are supported as two groups:
+
+**zones:** to list / describe `Kubernetes` topology zones and `Ceph` zones as configured
+
+**critical services:** to list/describe critical services defined in the RMS ConfigMap
+
+The sub commands for each of the groups are used to list, describe and update resources associated with the groups.
+
+RRS `craycli` commands
+
+```bash
+    cray rrs zones list
+    cray rrs zones describe 
+    cray rrs criticalservices list
+    cray rrs criticalservices describe
+    cray rrs criticalservices update --from-file <file-path>
+```
+
+Please refer https://github.com/Cray-HPE/docs-csm/blob/CASM-5612/operations/rack_resiliency/rack_resiliency_cli.md for more information on viewing/managing zones (`Kubernetes` and `Ceph`) and critical service via RR CLI.
+
+## Steps to achieve RR
+
+### Install/ Upgrade

```

Comment:
This is the only subsection -- shouldn't there also be one for enabling it later?
If it cannot be enabled except at install/upgrade time, that should be stated explicitly (probably both at the top of this document, and here). And in that case, you don't need this section header at all, since it's the only case.